lib/scrabble.rb

```diff
@@ -1,9 +1,33 @@
 class Scrabble
 
-  def score(word)
-    1
+
+
+  def score(word = nil)
+    score = 0
+    if word != nil
+      individual_letters = word.split('')
+      individual_letters.each do |letter|
+        point_value = point_values[letter.upcase]
+        score = score.to_i + point_value.to_i
+      end
+    end
+    score.to_i
+  end
+
+  def score_with_multipliers(word = nil, letter_multipliers)
+
+    score = 0
+      individual_letters = word.split('')
```

Comment:
Since this isn't code within an if/else or loop or enum, it doesn't need to be indented.